EIPS/eip-2348.md

```diff
@@ -0,0 +1,228 @@
+---
+eip: 2348
+title: Validated EVM Contracts
+author: Danno Ferrin (@shemnon)
+discussions-to: https://ethereum-magicians.org/t/eip-2348-validated-evm-contracts/3756
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-11-01
+requires: 1702
+---
+
+<!--
+HTML checker won't let me reference unpublished EIPs, namely 2327, 1707, 1712
```

Comment:
You can still put these into a references section or into the rationale.

Reply:
They are referenced alongside their ideas in the specification body.  I would have preferred to also have them in the header.

Reply:
You could also persuade #1707 and #1712 to fix the PRs for draft status :wink:

Reply:
1707 is abandoned in the PR text and getting wei to publish his EIPs in this repo is problematic at the moment, so I have links to the PRs.  

There is also the issue that they both need some minor revisions.  1707 needs to have the reference to WASM removed as that is out of scope of the "minimum required changes", and 1712 does not have accomidation for future evolution and only references `PUSHn` in the multibyte list, as well as not accounting for `BEGINDATA`.  So I could not reference those EIPs as the text currently stands.  And getting Wei to update the texts at the moment is blocked for reasons orthogonal to the content of those EIPs.